liferay/headless.py

```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
-                          'and in this ticket, fill the bug section.' \
-                          '\r\nRemember to link the bug (if you discover it) with the Story ticket.' \
-                          '\r\n{code:java}' \
-                          '\r\n*{color:#14892c}PASSED{color}* / *{color:#d04437}FAILED{color}* / *{' \
-                          'color:#59afe1}BLOCKED{color}* Manual Testing following the steps in the description.' \
-                          '\r\n' \
-                          '\r\n*Verified on:*' \
-                          '\r\n*Environment*: localhost' \
-                          '\r\n*Github*: https://github.com/liferay/liferay-portal.git' \
-                          '\r\n*Branch*: master' \
-                          '\r\n*Bundle*: Liferay DXP' \
-                          '\r\n*Database*: MySQL 5.7.22' \
-                          '\r\n*Last Commit*: ? ' \
-                          '\r\n' \
-                          '\r\n|| Test Scenarios || Test Result ||' \
-                          '\r\n| |*{color:#14892c}PASSED{color}* / *{color:#d04437}FAILED{color}* / *{' \
-                          'color:#59afe1}BLOCKED{color}*|' \
-                          '\r\n| |*{color:#14892c}PASSED{color}* / *{color:#d04437}FAILED{color}* / *{' \
-                          'color:#59afe1}BLOCKED{color}*|' \
-                          '\r\n...' \
-                          '\r\n{code}' \
-                          '\r\n*Bugs:*' \
-                          '\r\n (/)- PASS' \
-                          '\r\n (!)- To Do' \
-                          '\r\n (x)- FAIL' \
-                          '\r\n * *Impeditive:*' \
-                          '\r\n||Ticket||Title||' \
-                          '\r\n|?|?|' \
-                          '\r\n * *Not Impeditive:*' \
-                          '\r\n||Ticket||Title||' \
-                          '\r\n|?|?|'
-            subtask_test_validation = initialize_subtask_test_validation(story, components, description)
-            child = jira.create_issue(fields=subtask_test_validation)
-            print("* Created sub-task: " + child.key)
-
-    print("Subtasks for Headless team are up to date")
+    stories_with_test_creation_subtask = jira.search_issues('filter=54996')
+    for story in stories_with_test_creation_subtask:
+        for subtask in story.fields.subtasks:
+            summary = subtask.fields.summary
+            key = subtask.key
+            test_creation_subtask_details = jira.search_issues("key =" + key)
+            assignee = test_creation_subtask_details[0].fields.assignee
+            if summary == 'Test Scenarios Coverage | Test Creation':
+                print("Updating "+key+" ...")
+                if subtask.fields.status.name == "Open":
+                    description = '*Output*' \
+                                  '\r\n # Our table with the Test scenarios/test cases to be validated in the ' \
```

Comment:
It think it's better to add the \r\n at the end of the previous line so it's easier o read